pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -502,6 +516,36 @@ func (b *OFBridge) AddOFEntriesInBundle(addEntries []OFEntry, modEntries []OFEnt
 	return nil
 }
 
+func (b *OFBridge) SubscribePacketIn(reason uint8, ch chan *ofctrl.PacketIn) error {
+	_, exist := b.pktConsumers.Load(reason)
+	if exist {
+		return fmt.Errorf("packetIn reason %d already exists", reason)
+	}
+	b.pktConsumers.Store(reason, ch)
+	return nil
+}
+
+// AddTLVMap adds a TLV mapping with OVS field tun_metadataX. The value loaded in tun_metadataX is transported
+// by Geneve header with the specified <optClass, optType, optLength>. The value of OptLength is multiple of 4.
+// The value loaded into tun_metadataX is shorter the length.
+func (b *OFBridge) AddTLVMap(optClass uint16, optType uint8, optLength uint8, tunMetadataIndex uint16) error {
```

Comment:
@wenyingd since the comment is repeated between the interface declaration and the method implementation, could you make sure that the comments match? I suggested some fixes earlier for the comments and I think you applied them in the interface, but not here. This may apply to several functions.

Reply:
Yes, I have updated the comments as you suggested in the interface. I will remove the comments in the implementations.